eclipse/plugins/org.switchyard.tools.ui.bpmn2/META-INF/MANIFEST.MF

```diff
@@ -21,9 +21,9 @@ Require-Bundle: org.eclipse.ui,
  org.eclipse.emf.edit.ui,
  org.eclipse.emf.transaction,
  org.eclipse.bpmn2,
- org.eclipse.bpmn2.modeler.core;bundle-version="[1.1.0,1.4.0)",
- org.eclipse.bpmn2.modeler.ui;bundle-version="[1.1.0,1.4.0)",
- org.eclipse.bpmn2.modeler.runtime.jboss.jbpm;bundle-version="[1.2.3,1.4.0)",
+ org.eclipse.bpmn2.modeler.core;bundle-version="[1.1.0,1.5.0)",
```

Comment:
it's standard when changing major versions, not minor between 1.19 and 1.20 there should be no API break in theory.
but anyway I think that we are using specific target platform to avoid this kind of issue. That's fine.